### PR TITLE
Feature: Add mng api to create and delete service account

### PR DESCRIPTION
**Pull request by kengio**

### CODE REVIEW

**Review by mnaichuk — APPROVED** [review]

File: mngapi/barong/client.go [hunk 1]
```diff
@@ -20,3 +23,34 @@ func New(URL, jwtIssuer, jwtAlgo, jwtPrivateKey string) (*Client, error) {
...
+
+// CreateServiceAccount call barong management api to create new service account
+func (b *Client) CreateServiceAccount(params CreateServiceAccountParams) (*ServiceAccount, *mngapi.APIError) {
+	res, err := b.mngapiClient.Request(http.MethodPost, "service_accounts/create", params)
+	if err != nil {
+		return nil, err
+	}
+
+	serviceAccount := &ServiceAccount{}
+	_ = json.Unmarshal([]byte(res), serviceAccount)
```
Comment (calj) [inline mngapi/barong/client.go:35]: please handle all errors

File: mngapi/barong/client_test.go [hunk 2]
```diff
@@ -53,3 +54,87 @@ func TestCreateNewClient(t *testing.T) {
...
+		assert.NotNil(t, err)
+		assert.Equal(t, err.StatusCode, 422)
+		assert.Equal(t, err.Error, "User doesnt exist")
+		assert.Nil(t, serviceAccount)
+	})
+}
+
+func TestDeleteServiceAccountByUID(t *testing.T) {
+	t.Run("Success response", func(t *testing.T) {
+		client, _ := New(URL, jwtIssuer, jwtAlgo, jwtPrivateKey)
```
Comment (calj) [inline mngapi/barong/client_test.go:105]: also here in test it always good to retrive the error and assert:
```assert.NoError(t, err)```

File: mngapi/barong/client.go (hunk 1 above)
Comment (kengio) [reply mngapi/barong/client.go]: Added.

File: mngapi/barong/client_test.go (hunk 2 above)
Comment (kengio) [reply mngapi/barong/client_test.go]: Fixed.

**Pull request merged**